dlang.org.ddoc

```diff
@@ -242,16 +242,16 @@ $(SUBMENU
 )
 NAVIGATION_RESOURCES=
 $(SUBMENU
+    $(ROOT_DIR)acknowledgements.html, Acknowledgments,
+    https://wiki.dlang.org/Books, Books,
     $(ROOT_DIR)tools.html, D-Specific Tools,
```

Comment:
The problem is that dfix and dfmt aren't official tools (yet) and htod has to be downloaded separately. Some ideas that come to my mind:
1) We move the rdmd page over to the "DMD Manual" page and deprecate in favor of the wiki (the htod.html page looks very nice though)
2) We start to extend and update the tools page